src/messaging-component.js

```diff
@@ -52,7 +52,7 @@ const showExitModal = ({ cartRecovery }) => {
   return isRendered;
 };
 
-export const Messaging = (...args : $ReadOnlyArray<{ cartRecovery : { userId : string, beta? : boolean } }>) => {
+export const Messaging = (...args : $ReadOnlyArray<{| cartRecovery : {| userId : string, beta? : boolean |} |}>) => {
```

Comment:
this is updated by flow. I believe it sets strict definition of the type. I usually remove this change, as I do not mean to change existing classes

Reply:
fixed